packages/amplify-cli/src/index.ts

```diff
@@ -17,13 +17,14 @@ import { notify } from './version-notifier';
 import { EventEmitter } from 'events';
 import { rewireDeprecatedCommands } from './rewireDeprecatedCommands';
 import { ensureMobileHubCommandCompatibility } from './utils/mobilehub-support';
-import { postInstallInitialization } from './utils/post-install-initialization';
+import { deleteOldVersion } from './utils/win-utils';
 EventEmitter.defaultMaxListeners = 1000;
 
 // entry from commandline
 export async function run() {
   let errorHandler = (e: Error) => {};
   try {
+    deleteOldVersion();
```

Comment:
Is there a reason we always choose to delete the old version when running the CLI?

Reply:
it only needs to run on the command after upgrade, but to know that we would need to maintain some state about what the last command was. it's easier to just call this method every time and if the old version isn't there it's a noop